2023/04/wtchangdm/day4.rs

```diff
@@ -46,27 +47,16 @@ pub fn solve_part1(input: &[String]) -> u32 {
 
 pub fn solve_part2(input: &[String]) -> u32 {
     let mut count: HashMap<u32, u32> = HashMap::new();
-    let mut copies: HashMap<u32, usize> = HashMap::new();
-    let mut queue: VecDeque<u32> = VecDeque::new();
 
     input.iter().map(|line| Card::from(line)).for_each(|card| {
```

Comment:
Feel like we're doing mutable operations in `for_each`, which is fine but not ideal. I might recommend using `fold` or something else if we want to get a final result from `input` to a `HashMap`.

Regardless, looks good to me!

Reply:
fixed in #81!